torch/_C/__init__.pyi.in

```diff
@@ -118,6 +118,14 @@ class _LegacyVariableBase(object):
     ) -> None: ...
 
 # Defined in torch/csrc/jit/python/init.cpp
+class Future(object):
+  def __init(self) -> None: ...
```

Comment:
Is this intentional to use `__init` instead of `__init__`?

Reply:
I wonder why it wasn't detected by mypy